sdk/resources/arm-resources/package.json

```diff
@@ -17,17 +24,23 @@
     "isomorphic"
   ],
   "license": "MIT",
-  "main": "./dist/arm-resources.js",
-  "module": "./esm/resourceManagementClient.js",
-  "types": "./esm/resourceManagementClient.d.ts",
+  "main": "./dist/index.js",
+  "module": "./esm/index.js",
+  "types": "./esm/index.d.ts",
   "devDependencies": {
-    "typescript": "^3.6.0",
-    "rollup": "^1.18.0",
-    "rollup-plugin-node-resolve": "^5.2.0",
+    "@microsoft/api-extractor": "7.7.11",
+    "@rollup/plugin-commonjs": "11.0.2",
+    "@rollup/plugin-json": "^4.0.0",
+    "@rollup/plugin-multi-entry": "^3.0.0",
+    "@rollup/plugin-node-resolve": "^8.0.0",
+    "mkdirp": "^1.0.4",
+    "rollup": "^1.16.3",
     "rollup-plugin-sourcemaps": "^0.4.2",
-    "uglify-js": "^3.6.0"
+    "rollup-plugin-node-resolve": "^3.4.0",
+    "typescript": "~4.2.0",
+    "uglify-js": "^3.4.9"
   },
-  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/resources/arm-resources",
+  "homepage": "https://github.com/Azure/azure-sdk-for-js",
```

Comment:
this is weird. cc @sarangan12

Reply:
@deyaaeldeen This is a minor manual change, which is probably required only for the first time. I would not recommend doing any code level changes for this.